client/store/util.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * Returns a string representation of a sorted query object.
+ *
+ * @format
+ * @param {Object} query Current state
+ * @return {String}       Query Key
+ */
+
+export function getJsonString( query = {} ) {
```

Comment:
The docs here are weirdly formatted, I think from the prettier autoformatting – can you add `/** @format */` to the top of the file here, remove `@format` from this block, and remove the space after the docs so it's clear that it's a function doc?